src/inspectors/package/PHPPackageInspector.ts

```diff
@@ -0,0 +1,99 @@
+import { PackageInspectorBase } from './PackageInspectorBase';
+import { injectable, inject } from 'inversify';
+import { Types } from '../../types';
+import { keys } from 'lodash';
+import { DependencyType } from '../IPackageInspector';
+import { IFileInspector } from '../IFileInspector';
+
+@injectable()
+export class PHPPackageInspector extends PackageInspectorBase {
+  private fileInspector: IFileInspector;
+  private composerJson!: composerJSON;
+  private hasLockfileFile!: boolean;
+
+  constructor(@inject(Types.IFileInspector) fileInspector: IFileInspector) {
+    super();
+    this.fileInspector = fileInspector;
+  }
+
+  async init(): Promise<void> {
+    try {
+      this.debug('PHPPackageInspector init started');
+      const composerJsonString = await this.fileInspector.readFile('composer.json');
+      this.hasLockfileFile = await this.fileInspector.exists('composer.lock');
+      this.composerJson = JSON.parse(composerJsonString.replace(/\\/g, '/'));
+      this.packages = [];
+      this.addPackages(this.composerJson['require'], DependencyType.Runtime);
+      this.addPackages(this.composerJson['require-dev'], DependencyType.Dev);
```

Comment:
Please, use it as `this.composerJson.requireDev`. :)

Reply:
I guess this will not be possible as when the `composer.json` string is parsed as a JSON object, it still retails its keys in kebab-case. This can provide insight into what is really happening if we replace it with camelCase:

```js
> JSON.parse(`{"require-dev": "123"}`).requireDev
undefined
```

```js
> JSON.parse(`{"require-dev": "123"}`)['require-dev']
"123"
```

 I tried it on a repository and as expected, the keys which are in kebab-case are not present in `this.packages` when using the suggested camelCase notation.

Reply:
Oh, I see. Ok, then sorry for that. Can you just add some comment, that it must be in kebab case, please? :)

Reply:
Should I include it as a comment in `src/inspectors/package/PHPPackageInspector.ts`?

Reply:
Yes, please ❤️

Reply:
Made the changes :)